yoyodyne/models/pointer_generator.py

```diff
@@ -358,18 +358,75 @@ def forward(
                 h_features, c_features = self.init_hiddens(
                     len(batch), self.source_encoder.layers
                 )
-                predictions = self.decode(
-                    source_encoded,
-                    batch.source.mask,
-                    batch.source.padded,
-                    last_hiddens,
-                    self.teacher_forcing if self.training else False,
-                    features_enc=features_encoded,
-                    features_mask=batch.features.mask,
-                    target=batch.target.padded if batch.target else None,
-                )
+            predictions = self.decode(
+                source_encoded,
+                batch.source.mask,
+                batch.source.padded,
+                last_hiddens,
+                self.teacher_forcing if self.training else False,
+                features_enc=features_encoded,
+                features_mask=batch.features.mask,
+                target=batch.target.padded if batch.target else None,
+            )
         return predictions
 
+    def _get_loss_func(
+        self,
+    ) -> Callable[[torch.Tensor, torch.Tensor], torch.Tensor]:
+        """Returns the actual function used to compute loss.
+
```

Comment:
Could you add some brief documentation here that we're overriding the base version because we need NLLLoss for reasons? It could go into the docstring or even just a comment at line 381-382 or so.